components/pkg-export-helm/src/main.rs

```diff
@@ -66,6 +66,12 @@ fn export_for_cli_matches(ui: &mut UI, matches: &clap::ArgMatches) -> Result<()>
     Ok(())
 }
 
+lazy_static! {
+    pub static ref VERSION_HELP: String =
+        format!("Version of the chart to create (default: {{pkg_version}} if available, or {})",
+                chartfile::DEFAULT_VERSION);
+}
+
```

Comment:
Why is `lazy_static` needed here?

Reply:
because that's the only way to create a static string from two strings in Rust currently AFAIK. The `Arg::help()` we pass this to, expects a static string.